radar/techniques.js

```diff
@@ -1,4 +1,64 @@
 const content = [
+  {
+    name: 'Page Object Model',
+    ring: 'Adopt',
+    quadrant: 'Techniques',
+    isNew: 'FALSE',
```

Comment:
Dit is een nieuwe blip op de radar. 
```suggestion
    isNew: 'TRUE',
```